styles/globals.css

```diff
@@ -1 +1,108 @@
-Container {text-align: center;}
\ No newline at end of file
+.Game {
+  text-align: center;
+  background-color: #121212;
+  width: 100vw;
+  height: 100vh;
+  /* display: flex; */
+  color: white;
+}
+
```

Comment:
are we following figma style? could possibly lead to merge conflicts between toshi's and my global.css